wandb/sdk/internal/internal_api.py

```diff
@@ -1863,6 +1863,47 @@ def pop_from_run_queue(
         result: Optional[Dict[str, Any]] = response["popFromRunQueue"]
         return result
 
+    @normalize_exceptions
+    def get_run_queue_item(
+        self, entity_name: str, queue_name: str, run_queue_item_id: str
+    ) -> Optional[Dict[str, Any]]:
+        query = gql(
+            """
+        query RunQueueItem($entityName: String!, $queueName: String!, $itemId: ID!) {
+            project(name: "model-registry", entityName: $entityName) {
+                id
+                runQueue(name: $queueName) {
+                    id
+                    runQueueItem(id: $itemId) {
+                        id
+                        createdAt
+                        runSpec
+                        state
+                        priority
+                        associatedRunId
+                        launchAgentId
+                    }
+                }
+            }
+        }
+        """
+        )
+        response = self.gql(
+            query,
+            variable_values={
+                "entityName": entity_name,
+                "queueName": queue_name,
+                "itemId": run_queue_item_id,
+            },
+        )
+        print("========")
+        print(response)
+        print("========")
```

Comment:
debug tracks 👀  I guess not a big deal since we'll have to clear out a lot of debug prints before shipping LA2 as a whole